creme/proto/robust_soft_learning_vector_quantization.py

```diff
@@ -0,0 +1,388 @@
+import math
+
+import numpy as np
+
+from sklearn.utils import validation
+from sklearn.metrics import euclidean_distances
+from sklearn.utils.multiclass import unique_labels
+
+from creme.base import Classifier
+from creme.utils.skmultiflow_utils import check_random_state
+from creme.utils import dict2numpy
+
+
+class RobustSoftLearningVectorQuantization(Classifier):
+    """Robust Soft Learning Vector Quantization for Streaming and Non-Streaming Data.
+
+    By choosing another gradient descent method the Robust Soft Learning Vector Quantization
+    (RSLVQ) method can be used as an adaptive version.
+
+    Parameters
+    ----------
+    prototypes_per_class
+        Number of prototypes per class.
+    initial_prototypes: array-like, shape =  , optional
```

Comment:
No need for the type hint.

Reply:
My bad, this was not supposed to be there. I went back and fixed other minor issues with the documentation.